tests/rkt_fetch_test.go

```diff
@@ -147,10 +147,6 @@ func TestImplicitFetch(t *testing.T) {
 			t.Fatalf("Cannot exec rkt: %v", err)
 		}
 
-		if err := child.Expect(foundMsg); err != nil {
-			t.Fatalf("Expected %q but not found: %v", foundMsg, err)
-		}
-
```

Comment:
Moar flags! I'm not sure about that, why would a user want to force a download if the ETag is the hash of the image?